### ISSUE TITLE
Exclude generated columns from the insert type

### ISSUE THREAD

**Issue by yannxaver** [supabase/supabase-js#706]
# Bug report

## Describe the bug

Postgres allows the creation of generated columns: https://www.postgresql.org/docs/current/ddl-generated-columns.html. Inserting a value into such a column throws an error: https://www.tutorialsteacher.com/postgresql/generated-always. Supabase includes these columns in its `table['Insert']` type which goes against the functionality of postgres.

## To Reproduce

- create generated column
- generate types

## Expected behavior

Exclude generated columns.

**Comment by juanvilladev** [comment]
Yeah, this would be amazing. I don't see any use case for having the generated columns there. They shouldn't even be queryable by the Supabase Client IMO.
